tsuru_graphite.yml

```diff
@@ -0,0 +1,47 @@
+- hosts: "{{ hosts_prefix }}-tsuru-graphite"
+  sudo: yes
+  vars:
+    graphite_time_zone: "UTC"
+    graphite_storage_schemas_default_retentions: "10s:3d,1m:60d"
+    graphite_storage_file_retention_days: "65"
+  pre_tasks:
+    - name: Update apt cache
+      apt: update_cache=yes cache_valid_time=86400
+  roles:
+    - nsg.graphite
+    - dmichel1.statsd
+  tasks:
+    - name: Configure statsD idlestats
+      lineinfile:
```

Comment:
I would suggest use templates for this configuration.

Inline edition is often error prone and potentially not idempotent as it depends in the current state!